app.js

```diff
@@ -22,6 +24,27 @@ app.use(express.static(path.join(__dirname, "/public")));
 // Initialising database
 initDB();
 
+// Session options
+const sessionOptions = {
+  secret: "sanamSecretCode",
+  resave: false,
+  saveUninitialized: true,
+  cookie: {
+    expires: Date.now() + 7 * 24 * 60 * 60 * 1000,
+    maxAge: 7 * 24 * 60 * 60 * 1000,
+    httpOnly: true,
+  }
+}
+app.use(session(sessionOptions));
+
+// Implementing flash
+app.use(flash());
+app.use((req, res, next) => {
+  res.locals.success = req.flash("success");
+  res.locals.error = req.flash("error");
+  next();
+})
+
```

Comment:
Can create a middleware file and use this there calling session middleware here is the example


```
import session from "express-session";
import * as dotenv from "dotenv";
dotenv.config();

const oneDay = 24 * 60 * 60 * 1000;

const sessionMiddleware = session({
  secret: process.env.SECRET_KEY,
  resave: false,
  saveUninitialized: false,
  cookie: { expires: new Date(Date.now() + oneDay) },
});

export default sessionMiddleware;
```

use it like...
`app.use(sessionMiddleware);`
in index.js


same thing can be done for flash.. that means you can create a middleware folder where you will have two files..
SessionMiddleWare
FlashMiddleWare


These are middlewares thats why keeping it in seperate folder and using it here keeps your code clean